### PR TITLE
S3T3 Display Cats

### PR DESCRIPTION
**Pull request by Tibereon**

### CODE REVIEW

**Comment by mallowmew** [comment]
# Do not merge before s3t1!

File: index.php [hunk 1]
```diff
@@ -1,5 +1,15 @@
 <?php
 
+function drawCats(array $cats) :string {
+    $stringyCats = '';
+    foreach($cats as $cat) {
+        $stringyCats .= '<div class="cat-image"><img src="' . $cat->image . '" alt="A cat"></div>';
```
Comment (MotNosniktaw) [inline index.php:6]: $cat->image could be $cat-imageURL? 
or even $this->imageURL?

File: index.php [hunk 2]
```diff
@@ -1,5 +1,15 @@
 <?php
 
+function drawCats(array $cats) :string {
+    $stringyCats = '';
```
Comment (MotNosniktaw) [inline index.php:4]: more "professional" name?

File: index.php [hunk 3]
```diff
@@ -1,5 +1,15 @@
 <?php
 
+function drawCats(array $cats) :string {
```
Comment (MotNosniktaw) [inline index.php:3]: if this a function, it should be unit-tested

**Pull request merged**